webpack.config.js

```diff
@@ -222,7 +222,7 @@ module.exports = [
             library: [ 'JitsiMeetJS', 'app', 'effects' ],
             libraryTarget: 'window'
         }),
-        performance: getPerformanceHints(1 * 1024 * 1024)
+        performance: getPerformanceHints(2 * 1024 * 1024)
```

Comment:
😱 🚓 🚓 🚓  How come we are doubling the size? What causes this need for the increase? I assume it's the TF import. Can you please check if there is a way to import the module in a way we don't double the bundle size?